query-algebrizer/src/cc.rs

```diff
@@ -69,7 +69,7 @@ pub enum DatomsColumn {
 
 impl DatomsColumn {
     pub fn as_str(&self) -> &'static str {
-        use DatomsColumn::*;
+        use DatomsColumn::{Attribute, Entity, Tx, Value, ValueTypeTag};
```

Comment:
I don't think we should expand this style, and will bring it up in the meeting.  The whole point is to avoid being verbose; we're trading `match _ { X::Y => ; X::Z => ; }` for `use X::{Y, Z}; match _ { X => ; Z => }`, which isn't a win to my eyes.